docs/_config.yml

```diff
@@ -16,8 +16,8 @@ include:
 # of Spark, Scala, and Mesos.
 SPARK_VERSION: 2.4.2-SNAPSHOT
 SPARK_VERSION_SHORT: 2.4.2
-SCALA_BINARY_VERSION: "2.11"
-SCALA_VERSION: "2.11.12"
+SCALA_BINARY_VERSION: "2.12"
```

Comment:
Note: these values only affect the docs, and whether we render things like "org.apache.spark:foo_2.12" in docs. It doesn't affect the code that is built.